d4science-base/Dockerfile

```diff
@@ -19,8 +19,6 @@ RUN curl https://raw.githubusercontent.com/kadwanev/retry/master/retry \
 USER $NB_UID
 
 RUN mamba install -y -q \
-        r-plotly \
```

Comment:
These two were causing issues... They are R related so now handled in the R image